CONTRIBUTING.md

```diff
@@ -49,3 +49,19 @@ This project follows
 
 We pledge to maintain an open and welcoming environment. For details, see our
 [code of conduct](https://dart.dev/code-of-conduct).
+
+## Tests
+
+Packages `native_assets_cli`, `native_assets_builder`, and `native_toolchain_c`
+roll into the Dart SDK and flutter_tools. The tests of these packages are run
```

Comment:
```suggestion
roll into the Dart SDK. The tests of these packages are run
```

Reply:
They do roll into the flutter SDK as well. Not via DEPS but via us updating published dependencies. But in Flutter the tests are not run, only the tests that do `flutter create template=package_ffi` are run, which then exercise the code.

Reply:
Then best document that behavior as well, if it is relevant. Otherwise don't mention it.

Reply:
I guess that doesn't actually influence the tests, so maybe we should leave it out then. (`## Test` is the header.)